modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -6,8 +6,12 @@
 [id="configuring-linuxptp-services-as-boundary-clock_{context}"]
 = Configuring linuxptp services as boundary clock
 
-The PTP Operator adds the `PtpConfig.ptp.openshift.io` custom resource definition (CRD) to {product-title}.
-You can configure the `linuxptp` services (`ptp4l`, `phc2sys`) by creating a `PtpConfig` custom resource (CR) object.
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`) as boundary clock by creating a `PtpConfig` custom resource (CR) object.
+
+[NOTE]
+====
+The below `PtpConfig` resource configures PTP fast events by setting values for `ptp4lOpts`, `ptp4lConf` and `ptpClockThreshold`.
```

Comment:
```suggestion
The `PtpConfig` resource configures PTP fast events by setting values for `ptp4lOpts`, `ptp4lConf` and `ptpClockThreshold`.
```
Don't think below here is needed either. But ignore if it changes the meaning of the sentence.